tests/phpunit/Mutator/Util/MutatorsGeneratorTest.php

```diff
@@ -98,7 +98,7 @@ public function test_profiles_can_be_ignored(): void
         ]);
         $mutators = $mutatorGenerator->generate();
 
-        $this->assertCount(self::$countDefaultMutators - count(MutatorProfile::BOOLEAN), $mutators);
+        $this->assertCount(self::$countDefaultMutators - count(\Infection\Mutator\ProfileList::BOOLEAN_PROFILE), $mutators);
```

Comment:
Hm not sure why this is happening occasionally but I'm changing this test in my next PR anyway